src/chart.zoom.js

```diff
@@ -198,7 +198,13 @@ function doZoom(chartInstance, zoomX, zoomY, center, whichAxes) {
 		chartInstance.update(0);
 
 		if (typeof zoomOptions.onZoom === 'function') {
-			zoomOptions.onZoom();
+			zoomOptions.onZoom({
+				chartInstance: chartInstance,
+				zoomX: zoomX,
```

Comment:
What are `zoomX` and `zoomY`?  (I took over maintaining this repo, so don't know all the code). Can we remove them and get by with just `center`? If not, can we rename or document to clarify?

Reply:
I looked into what this is doing and am proposing to rename to `percentZoomX` and `percentZoomY` in https://github.com/chartjs/chartjs-plugin-zoom/pull/192